### PR TITLE
Feature/confloader

### PR DESCRIPTION
**Pull request by ben-en**
Switches server usage from agparse to confloader. See server docs for command usage.

https://github.com/Outernet-Project/oftps/blob/cc44664f82f65a1d6f658c9a0801e9ebd2e3d507/docs/html/_sources/server.txt

### CODE REVIEW

File: docs/html/_sources/intro.txt [hunk 1]
```diff
@@ -7,7 +7,7 @@ Server
 
 To run the server with dummy values you can do::
 
-    ./oftps/server.py db.sqlite tests/test_files/dummy.key tests/test_files/dummy.cert -l tests/test_files/user.list
+    ./oftps/server.py --conf oftps/config.ini
```
Comment (foxbunny) [inline docs/html/_sources/intro.txt]: :+1: for documentation update

File: oftps/server.py [hunk 2]
```diff
@@ -153,17 +153,14 @@ def start_server(db, key, cert, dir, userfile):
...
-    parser.add_argument('db', metavar='DB', help='database file to use')
-    parser.add_argument('key', metavar='KEY', help='TLS key file')
-    parser.add_argument('cert', metavar='CERT', help='TLS certificate')
-    parser.add_argument('--userlist', '-l', metavar='USERLIST', help='file to '
-                        'fetch the user list from', default='user.list')
-    parser.add_argument('--workingdir', '-w', metavar='WORKING', default='.',
-                        help='directory to broadcast from')
-    args = parser.parse_args()
-    start_server(args.db, args.key, args.cert, args.workingdir, args.userlist)
+    confloader.get_config_path(default='config')
```
Comment (foxbunny) [inline oftps/server.py]: Return value from `get_config_path()` should be the arg that was passed, and it should be passed on to `from_file()`.

File: oftps/server.py [hunk 3]
```diff
@@ -153,17 +153,14 @@ def start_server(db, key, cert, dir, userfile):
...
+    conf = confloader.ConfDict.from_file('config.ini', defaults={
```
Comment (foxbunny) [inline oftps/server.py]: This call always loads `config.ini`.

Comment (ben-en) [inline oftps/server.py]: Fixed in updated diff.

**Pull request merged**